application/comit_node/src/http_api/rfc003/handlers/get_swap.rs

```diff
@@ -61,28 +64,40 @@ pub fn handle_get_swap<T: MetadataStore<SwapId>, S: StateStore>(
 }
 
 #[derive(Debug, Serialize)]
-#[serde(bound = "AL: Ledger, BL: Ledger")]
-pub struct GetSwapResource<AL: Ledger, BL: Ledger> {
-    parameters: SwapParameters,
+#[serde(
+    bound = "Http<AL>: Serialize, Http<BL>: Serialize, Http<AA>: Serialize, Http<BA>: Serialize,\
```

Comment:
❗️ also this

Reply:
And this I figured out with @thomaseizinger's help :stuck_out_tongue: